.clang-tidy

```diff
@@ -23,5 +33,7 @@ CheckOptions:
     value:        'assert,Assert,Assertion'
   - key:          'modernize-use-emplace.ContainersWithPushBack'
     value:        'std::vector;std::deque;std::list;SCP_vector;SCP_deque;SCP_list'
+  - key:          'readability-braces-around-statements.ShortStatementLines'
+    value:        '2' # Avoid flagging simple if (...) return false; statements
```

Comment:
This option should not be included.  It affected many lines in my PRs, and if I'm not mistaken it affected a few lines in your PR too.  I specifically reverted this in #2396.

Reply:
Maybe the value is too low. I will increase it to 4. That should make sure that only the bad cases are flagged by this.